suites/pacific/rbd/tier-2_rbd_mirror_regression.yaml

```diff
@@ -160,6 +160,17 @@ tests:
       polarion-id: CEPH-9501
       desc: Verify that image deleted at primary site updated at secondary
 
+  - test:
+      name: test image delete from secondary after multiple promote and demote
+      module: test_83574741.py
+      clusters:
+        ceph-rbd1:
+          config:
```

Comment:
Add config say repeat_count which will have the number of times promote and demote should be tried before deleting the images.